.github/workflows/check-pr-title.yaml

```
@@ -0,0 +1,31 @@
+name: "Check PR title"
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  check-title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if the PR title is well dressed
+        env:
+          PR_TITLE: ${{ github.event.pull_request.title }}
+          CONV: '(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(\([\w\-\.]+\))?!?'
+          ROX: '(ROX-[0-9]+,)*ROX-[0-9]+'
+          TEXT: ': .+'
+          LENGTH: '70'
+        run: |
+          # Either conventional or ROX-nnn prefix followed by ': ' and random text:
+          REGEX="(($CONV)|($ROX))($TEXT)"
+          if ! printf %s "$PR_TITLE" | grep --perl-regexp --line-regexp "$REGEX"
+          then
+            echo "::error::The PR title doesn't follow the convention."
+            exit 1
+          fi
+
+          if [ "$(printf %s "$PR_TITLE" | wc --chars)" -gt "$LENGTH" ]
+          then
+            echo "::error::The PR title is longer than $LENGTH characters."
```

1. Let's change the message to motivate, not punish.
```suggestion
            >&2 echo "::error::Please shorten the PR title to be $LENGTH characters or less."
```
2. Would be nice to output the length of the current title so that users have a clue how much to cut.